backend-processing.ts

```diff
@@ -1,4 +1,5 @@
 import { resolve } from "dns";
+import { start } from "repl";
 
 const denv = require("dotenv").config();
```

Comment:
if calling `require("dotenv").config();` is the only thing needed to make it work then you don't need to assign the results to a variable. You can just call the function.

so instead of

```javascript
const denv = require("dotenv").config();
```

it would be

```javascript
require("dotenv").config();
```